src/components/VAccordion.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="accordion">
+    <div
+      class="accordion-header"
+      :class="{'accordion-header-active': visible}"
+      tabindex="0"
+      @click="open"
+      @keypress.enter.space="open"
+    >
+      <div class="accordion-header-wrapper">
+        <slot name="header" />
+      </div>
+      <div class="accordion-icon">
+        <simple-arrow-down v-if="active" />
+        <simple-arrow-up v-else />
+      </div>
+    </div>
+    <transition name="accordion">
+      <div
+        v-show="visible"
+        class="accordion-content"
+        :style="'max-height: '+height+'px'"
+      >
+        <slot name="content" />
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import SimpleArrowDown from '../components/icons/SimpleArrowDown'
+import SimpleArrowUp from '../components/icons/SimpleArrowUp'
+
+export default {
+  components: {
+    SimpleArrowDown,
+    SimpleArrowUp
+  },
+  props: {
+    openOnLoad: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      active: this.openOnLoad,
+      height: '500px'
+    }
+  },
+  computed: {
+    visible () {
+      return this.active
+    }
+  },
+  updated () {
+    this.height = this.$slots.content[0].elm.clientHeight
```

Comment:
Was trying to look this up in the Vue docs cause I was wondering if there was a reason about why it sometimes uses `el` and sometimes uses `elm` for elements. Finally found the section on the VNode Interface, where `.elm` is defined, in the API docs and... it's just a link to the source code.  I guess some mysteries will never be solved.

Reply:
$el is also used sometimes. this is the one that worked!

Reply:
Changing to the more up to date slot usage made .elm not work, but now $el works!